pages/common/crontab.md

```diff
@@ -13,3 +13,19 @@
 - Remove all cron jobs for the current user:
 
 `crontab -r`
+
+- Cron job definition format:
+
+`{{minute}} {{hour}} {{day_of_month}} {{month}} {{day_of_week}} {{command_to_execute}}`
+
+- Sample job which runs at 10:00 every day. * means any value:
+
+`0 10 * * * {{path/to/script.sh}}`
+
+- Sample job which runs every minute on the 3rd of April:
+
+`* * 3 4 * {{path/to/script.sh}}`
+
+- Sample job which runs at 02:30 every friday:
```

Comment:
We definitely should mention the range of values (Monday=1 Saturday=6, Sunday=0/7).

We should also have an example using a weekday name (can we use names for other entries, e.g. months?).